test/cmdlineTests/yul_verbatim/output

```diff
@@ -0,0 +1,26 @@
+
+======= yul_verbatim/input.yul (EVM) =======
+
+Pretty printed source:
+object "object" {
+    code {
+        {
+            sstore(verbatim_2_1("abc", 2, 0), 2)
```

Comment:
Potentially this naming scheme may be superior to the original suggestion of having the return stack count as first argument and deriving input stack count by the arguments.

However I wonder fi we can make it more easy to read, some ideas:
- `verbatim_i2_o1`
- `verbatim_in2_out1`, `verbatim_in0_out0`, ...
- `verbatim_2i_1o`
- `verbatim_2x1`

Reply:
This is also solved